src/creature.cpp

```diff
@@ -1737,6 +1742,11 @@ void Creature::set_part_frostbite_timer( const bodypart_id &id, int set )
     set_part_helper( *this, id, &bodypart::set_frostbite_timer, set );
 }
 
+void Creature::set_part_mut_drench( const bodypart_id &id, std::pair<water_tolerance, int> set )
```

Comment:
Same here.

Reply:
Yes but it doesn't sit well set_part_helper() that expect only one argument. I messing with the helper seemed more complicated

Reply:
Ah, right, the `part_helper`. Ok, nevermind then. 

But we probably need to have a discussion in the future regarding the `get_part_helper` and `set_part_helper`.